atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -94,26 +94,30 @@ def run(self):
         source_request = self.cachito_session.wait_for_request(source_request)
 
         remote_source_json = self.source_request_to_json(source_request)
-        remote_source_url = self.cachito_session.assemble_download_url(source_request)
-        remote_source_conf_url = remote_source_json.get('configuration_files')
-        remote_source_icm_url = remote_source_json.get('content_manifest')
-        self.set_worker_params(source_request, remote_source_url, remote_source_conf_url,
-                               remote_source_icm_url)
+        remote_sources = [
+            {
+                "build_args": None,
+                "configs": remote_source_json.get('configuration_files'),
+                "request_id": self.cachito_session._get_request_id(source_request),
+                "url": self.cachito_session.assemble_download_url(source_request),
+                "name": None,
+            }
+        ]
+        self.set_worker_params(source_request, remote_sources)
 
         dest_dir = self.workflow.source.workdir
         dest_path = self.cachito_session.download_sources(source_request, dest_dir=dest_dir)
 
         return {
             # Annotations to be added to the current Build object
-            'annotations': {'remote_source_url': remote_source_url},
+            'annotations': {'remote_source_url': remote_sources[0]['url']},
             # JSON representation of the remote source request
             'remote_source_json': remote_source_json,
             # Local path to the remote source archive
             'remote_source_path': dest_path,
         }
 
-    def set_worker_params(self, source_request, remote_source_url, remote_source_conf_url,
-                          remote_source_icm_url):
+    def set_worker_params(self, source_request, remote_sources):
```

Comment:
we don't need source_request anymore to get 'id'

as that id is also already in remote_sources

Reply:
updated